src/jit/jitconfigvalues.h

```diff
@@ -0,0 +1,193 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
> that sort of policy is not up to the JIT

To phrase this differently: all the JIT does is ask the EE for the value of a configuration point. Obtaining that value (and all of the associated policy) is the EE's responsibility.